package.json

```diff
@@ -66,7 +78,8 @@
 						"array"
 					],
 					"default": [
-						"php"
+						{ "scheme": "file", "language": "php" },
+						{ "scheme": "untitled", "language": "php" }
```

Comment:
changed this to the expanded values, and allowed either strings for the expanded values for the setting. (Still backwards compatible with old values.)